packages/commonwealth/client/scripts/state/api/communityStake/fetchEthUsdRate.ts

```diff
@@ -0,0 +1,29 @@
+import { useQuery } from '@tanstack/react-query';
+import axios from 'axios';
+import { ExternalEndpoints } from 'state/api/config';
+
+const ETH_USD_RATE_STALE_TIME = 10 * 60 * 1_000; // 10 min
```

Comment:
I don't have much background here, but is it safe to cache this for 10 mins? If a user buys a stake and the real ETH_USD value increases/decreases, do we use the cached value or the updated value?

Reply:
we call it pretty often so caching it for longer time makes sense. Also, this is also for displaying approximate USD value and I assume that the rate does not change significantly within 10 minutes.